### PR TITLE
add "No messages" as empty state message

### PR DESCRIPTION
**Pull request by alexander-demichev**
Add "No messages" and icon as empty state message

---
This change is [Reviewable](https://reviewable.io/reviews/ovirt/ovirt-web-ui/419)

### CODE REVIEW

**Comment by alexander-demichev** [comment]
https://github.com/oVirt/ovirt-web-ui/issues/386

File: src/components/VmUserMessages/index.js [hunk 1]
```diff
@@ -66,6 +66,26 @@ class VmUserMessages extends React.Component {
...
+    let messagesList = ''
+
+    if (messages.size !== 0) {
+      messagesList = (
+        <ul className={`list-group ${style['messages-list']}`}>
+          {messages.map(r => (<UserMessage key={r.time} record={r} id={`${idPrefix}-msg-${idCounter++}`} />))}
+        </ul>)
+    } else {
+      messagesList = (
+        <div className='blank-slate-pf '>
```
Comment (jniederm) [inline src/components/VmUserMessages/index.js]: Could you please add custom style rules `border` and `magin-bottom` for this element to remove unnecessary white space and border line?

image

File: src/intl/messages.js [hunk 2]
```diff
@@ -117,6 +117,7 @@ export const messages = {
     description: 'About application',
   },
   logOut: 'Log out',
+  noMessages: 'No messages',
```
Comment (jniederm) [inline src/intl/messages.js]: Please replace the the value by 
```
{
  message: 'No\u00A0messages'.
  description: '\u00A0 is code for unbreakable space, please use it in your translation',
}
```

**Comment by mareklibra** [comment]
Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/intl/messages.js, line 120 at r2](https://reviewable.io:443/reviews/ovirt/ovirt-web-ui/419#-KzELyBL0zjCD8G6eDD--r2-120:-KzO3Tj_jOo88dVITRqE:b-kdae0l) ([raw file](https://github.com/ovirt/ovirt-web-ui/blob/a906a4209b720d50c3a221da0a1eac480129eb7a/src/intl/messages.js#L120)):*
<details><summary><i>Previously, jniederm wrote…</i></summary><blockquote>

Please replace the the value by 
```
{
  message: 'No\u00A0messages'.
  description: '\u00A0 is code for unbreakable space, please use it in your translation',
}
```
</blockquote></details>

Please add the description for translators. It might not be obvious for them.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ovirt/ovirt-web-ui/419)*

**Comment by jniederm** [comment]
GoodnessSquad participants don't have engine available so addressing comments is not expected. Commit reused in pull #435.

**Pull request closed without merging**